eclass/rpm.eclass

```diff
@@ -90,11 +94,15 @@ rpm_src_unpack() {
 
 # @FUNCTION: rpm_spec_epatch
 # @USAGE: [spec]
+# @DEPRECATED: none
 # @DESCRIPTION:
 # Read the specified spec (defaults to ${PN}.spec) and attempt to apply
 # all the patches listed in it.  If the spec does funky things like moving
 # files around, well this won't handle that.
 rpm_spec_epatch() {
+	# no epatch in EAPI 7 and later
+	[[ ${EAPI} == [56] ]] || die "${FUNCNAME} is banned in EAPI ${EAPI}"
```

Comment:
As a minor hint for users:
```suggestion
	[[ ${EAPI} == [56] ]] || die "${FUNCNAME} is banned in EAPI ${EAPI}, use eapply"
```

Reply:
No ebuild is using that function, and nobody noticed that it was broken in EAPI 7.
If someone will need it again, it may be revived with eapply. Until then, I'd rather not add any incomplete instructions how to replace it.